### DISCUSSION TITLE
New _recid

### DISCUSSION THREAD

**Discussion by djbbiz** [cortexrd/Knack-Toolkit-Library#158] [Q&A] [answered]
Hey Norm, I like the ability to add the record ID to the object as often I have to look it up while doing API work. But that being said, usually my users do not need to see it. Can I combine this function with another keyword to hide the column from view to regular users so say the Developer can see the column in the grid with the Record ID but the regular Users cannot?

**Comment by CSWinnall** [comment]
Hi Dean

I don't see why you couldn't use hide colums

_hc=[recId], [ktlRoles, !Developer]

Craig

**Reply by CSWinnall** [reply, marked as answer]
Sorry I should have said recId is the name of column
